runtime/runtime/src/congestion_control.rs

```diff
@@ -335,9 +408,97 @@ impl ReceiptSinkV2<'_> {
         self.own_congestion_info.add_receipt_bytes(size)?;
         self.own_congestion_info.add_buffered_receipt_gas(gas)?;
 
+        if receipt.queue_metadata_version() == Some(QueueMetadataVersion::OutgoingBufferMetadataV0)
+        {
+            self.outgoing_buffer_metadatas.on_receipt_buffered(shard, size);
+        }
```

Comment:
nit: Maybe make it into a match so that when new versions are added the compiler will remind us to handle it.

Reply:
Good point, will do :+1: